### ISSUE TITLE
[V2] Build fails with shader error undeclared identifier 'IN' at UnityCG.cginc(766) in SPS Instanced

### ISSUE THREAD

**Issue by BlackBoxSimulations** [Unity-Technologies/PostProcessing#330]
##### What happened?
Build fails with error:
Shader error in 'Sprites/Mask': undeclared identifier 'IN' at /Unity_2017.2/Editor/Data/CGIncludes/UnityCG.cginc(766) (on d3d11)

##### Post-processing stack version (v1, v2)?
v2 (xr-update)

##### Unity version, operating system, target platform (standalone windows, mac, iOS, PS4...)?
Unity 2017.2.0
Windows 10
Standalone Windows (Single Pass Stereo Instanced) (d3d11)

**Comment by robbiesri** [comment]
Does this repro on the vanilla v2 branch?  This doesn't look specific to the xr-update branch...

Also, stereo instancing support isn't in the xr-update branch yet :/

**Comment by davidhaley** [comment]
I received a similar error in 2017.3.0b5. I am using The Lab Renderer ( it worked fine with 2017.2.0b2 ):

"Shader error in 'Sprites/Mask': undeclared identifier 'IN' at /Program Files/Unity 2017.3.0b5/Editor/Data/CGIncludes/UnityCG.cginc(775) (on d3d11)"

**Comment by njbrown09** [comment]
Any ETA on this? I cant even compile my game

**Comment by BlackBoxSimulations** [comment]
Ok, so I tried this with sps instanced off (just sps) and still received the same error message.

I then switched from the xr-update branch to the vanilla branch and tried again with standard SPS. The error persists.

Shader error in 'Sprites/Mask': undeclared identifier 'IN' at /Unity_2017.2/Editor/Data/CGIncludes/UnityCG.cginc(766) (on d3d11)

Compiling Vertex program with ETC1_EXTERNAL_ALPHA PLUGIN_STEREOX
Platform defines: UNITY_ENABLE_REFLECTION_BUFFERS UNITY_USE_DITHER_MASK_FOR_ALPHABLENDED_SHADOWS UNITY_PBS_USE_BRDF1 UNITY_SPECCUBE_BOX_PROJECTION UNITY_SPECCUBE_BLENDING UNITY_ENABLE_DETAIL_NORMALMAP SHADER_API_DESKTOP UNITY_LIGHT_PROBE_PROXY_VOLUME

**Comment by BlackBoxSimulations** [comment]
Quick update,

I created a clean project, dropped in the V2 xr-update and I was able to complete a build using SPS (not instanced) after I removed DXD9 from the api compatibility list in player settings.

The only difference was that I used the latest xr-update branch.

**Comment by robbiesri** [comment]
The more I see, the less I think this has anything to do with the Post-processing Stack...

**Comment by johnnyjr** [comment]
Can confirm it has nothing to do with the Post-processing stack. I got the same error, and since this is one of the only threads I found discussing it I thought I should provide the solution here. 
The errror "Shader error in 'Sprites/Mask': undeclared identifier 'IN' at /Program Files/Unity 2017.2.0b2/Editor/Data/CGIncludes/UnityCG.cginc(764) (on d3d11)" is due to Unity compiling shaders with PLUGIN_STEREOX - which is VRWorks. Removing VRWorks from the project solved it for me.
